### DISCUSSION TITLE
Additional Permissions or a in-game command toggle

### DISCUSSION THREAD

**Discussion by itsBeaco** [PurpurMC/Purpur#1088] [__Completed__] [answered]
Hey,

Would it be possible to add a permission and/or an in-game command to toggle on/off shift-right-click-repairs-mending?

Most players really like it but some players would like to be able to turn it off when they are shift bridging

If it's possible, then amazing, if not then no worries. Thank you in advance!

**Comment by BillyGalbreath** [comment]
What do you mean shift bridging?

**Comment by encode42** [comment, marked as answer]
Implemented via the `purpur.mending_shift_click` permission in build 1820.
